tinkerbell/stack/templates/nginx.yaml

```diff
@@ -139,6 +139,10 @@ metadata:
     app: {{ .Values.stack.name }}
   name: {{ .Values.stack.name }}
   namespace: {{ .Release.Namespace | quote }}
+  annotations:
+  {{- with .Values.stack.service.annotations | default dict }}
+    {{ toYaml . | nindent 4 }}
```

Comment:
Are you confident this works properly? I would've thought you'd want a preceding `-` to get rid of the whitespace up to `annotations:`.

```
    {{- toYaml . | nindent 4 }}
```